include/aspect/material_model/utilities.h

```diff
@@ -259,6 +266,82 @@ namespace aspect
                           const bool interpol,
                           const MPI_Comm &comm);
         };
+
+        /**
+        * This class reads in an entropy-pressure material table and looks up material
+        * proporties for the given entropy and pressure.
+        */
+        class EntropyReader
+        {
+          public:
+
+            /**
+             * Read the table.
+             */
+            void
+            initialize(const MPI_Comm comm,
+                       const std::string data_directory,
+                       const std::string material_file_name);
+
+            /**
+             * Reads the specific heat for a given entropy and pressure.
```

Comment:
`Reads` is maybe not the correct work. What about:
```suggestion
             * Returns the specific heat for a given entropy and pressure.
```
Same for all the other functions below.